torch/csrc/jit/register_prim_ops.cpp

```diff
@@ -430,6 +461,15 @@ RegisterOperators reg2({
     Operator("aten::add(float[] a, float[] b) -> float[]", listAdd<Shared<DoubleList>, double>),
     Operator("aten::add(Tensor[] a, Tensor[] b) -> Tensor[]", listAdd<Shared<TensorList>, at::Tensor>),
 
+    Operator(
```

Comment:
I did this default argument thing to be consistent with the signature of `aten::slice`, but it seems kinda gross? Let me know if there's a better way.